CommonTools/PileupAlgos/plugins/PuppiProducer.cc

```diff
@@ -355,12 +353,10 @@ void PuppiProducer::fillDescriptions(edm::ConfigurationDescriptions& description
   desc.add<bool>("UseFromPVLooseTight", false);
   desc.add<bool>("UseDeltaZCut", true);
   desc.add<double>("DeltaZCut", 0.3);
-  desc.add<double>("PtMaxCharged", 0.);
-  desc.add<double>("EtaMaxCharged", 0.);
+  desc.add<double>("PtMaxCharged", 7000.);
```

Comment:
As I said, 7000 cannot guarantee that all particles, even the pathological ones, are kept